tokio-util/src/time/delay_queue.rs

```diff
@@ -66,9 +66,13 @@ use std::task::{self, ready, Poll, Waker};
 /// Capacity can be checked using [`capacity`] and allocated preemptively by using
 /// the [`reserve`] method.
 ///
+/// # Cancel safety
```

Comment:
```suggestion
/// #  Cancellation safety
```

I personally prefer the "Cancellation safety", not a blocker.